dev/build/windows/makecoq_mingw.sh

```diff
@@ -1955,7 +1955,13 @@ function make_addon_gappa {
   if build_prep_overlay gappa_plugin ; then
     installer_addon_section gappa "Gappa plugin" "Coq plugin for the Gappa tool" ""
     log1 autoreconf
+
+    # coqc - config prints broken paths
+    OLD_OCAMLFIND="$OCAMLFIND"
+    export OCAMLFIND=ocamlfind
     logn configure ./configure
+    export OCAMLFIND="$OLD_OCAMLFIND"
```

Comment:
If you do not want to play this dance with environment variables, just pass it on the command line:

    logn configure ./configure OCAMLFIND=ocamlfind

Reply:
thanks, I'll do. Unfortunately I did not manage to test your new commit yet, since CoqIDE is broken.